provisioning-telephony/config.yml

```diff
@@ -14,53 +14,55 @@ steps:
     * Have a publicly routable IP address or host name
       * Carriers must fully support Record-Routing as required by RFC 3261 to ensure proper routing of sequential SIP requests.
     
-    For more information about carrier requirements, check Genesys Resource Center article [Carrier requirements for BYOC Cloud](https://help.mypurecloud.com/articles/carrier-requirements-byoc-cloud/).
+    The Org must also have the BYOC Cloud feature enabled. For more information about carrier requirements, check the Genesys Resource Center article [Carrier requirements for BYOC Cloud](https://help.mypurecloud.com/articles/carrier-requirements-byoc-cloud/).
     
     To successfully run the code, a JSON file is needed to provide the necessary input. In this tutorial, the JSON file is titled input-template.json.
 
 - title: JSON Files and Variables
   content: |
-    This file is used as substitute to input text box. This is where the user enters all the necessary details. Key value pair has its corresponding description. Please make sure to replace all the value of keys from description to actual data and fill all the required fields to proceed. 
+    This file is used as substitute to input text boxes. This is where the user enters all of the necessary details. Each key value pair has a corresponding description. Ensure all values are entered where required, or left blank where entry is optional. Any phone number entries should use the E.164 format. 
     
-    Please refer to [BYOC Cloud public SIP IP addresses](https://help.mypurecloud.com/articles/byoc-cloud-public-sip-ip-addresses/) for instance or Public SIP Addresses.  
+    Please refer to your carrier for the possible SIP Endpoints as a list of IP Addresses or CIDR. See the example below. 
+
+    Sample: ['54.172.60.0/30 ','54.244.51.0/30','54.171.127.192/30','35.156.191.128/30']
```

Comment:
There's an extra space in the first value. Also, put inline literal values in single backticks, i.e.:

```md
Sample: `['54.172.60.0/30','54.244.51.0/30','54.171.127.192/30','35.156.191.128/30']`
```

Reply:
Done eliminating the extra space and adding backticks.